### ISSUE TITLE
Don't fetch git history when initialize Travis-CI

### ISSUE THREAD

**Issue by gloomy-ghost** [EFForg/https-everywhere#7544]
There is already a limitation of initial cloning:

https://travis-ci.org/EFForg/https-everywhere/jobs/175365628#L85
`$ git clone --depth=50 https://github.com/EFForg/https-everywhere.git EFForg/https-everywhere`

But then it fetches the PR without limitation:

https://travis-ci.org/EFForg/https-everywhere/jobs/175365628#L95
```
$ git fetch origin +refs/pull/6921/merge:
remote: Counting objects: 380576, done.
remote: Compressing objects: 100% (132600/132600), done.
remote: Total 380576 (delta 256689), reused 371093 (delta 247437), pack-reused 0
Receiving objects: 100% (380576/380576), 133.15 MiB | 20.94 MiB/s, done.
Resolving deltas: 100% (256689/256689), completed with 9113 local objects.
From https://github.com/EFForg/https-everywhere
 * branch            refs/pull/6921/merge -> FETCH_HEAD
```

It usually hangs some times to progress resolving, and I think that it's unnecessary to fetch histories at auto-testing. Can we use `--depth=1` here?

Pinging @Hainish

**Comment by Hainish** [comment]
Good catch @gloomy-ghost.  I've opened a PR to fix this: https://github.com/EFForg/https-everywhere/issues/7544

**Comment by gloomy-ghost** [comment]
Cool, but did you mean #7553?

**Comment by Hainish** [comment]
Yes, that's what I meant :)

**Comment by gloomy-ghost** [comment]
Closed by #7553
